notebooks/book1/02/centralLimitDemo.ipynb

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.            plt.ylabel("$mean$")
</pre><p>For xlabel and ylabel, we sould search about this, I think at x_label we can use "bins".  And for y-label I'm not sure we can use "mean" or something different word given that this is histogram plot not the pdf plot. One way is to check any histogram plot in mml book.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
The file has been updated. Please have a look and let me know